app/build.gradle

```diff
@@ -163,23 +162,26 @@ dependencies {
     implementation "io.reactivex.rxjava2:rxandroid:$rxAndroidVersion"
     implementation "io.reactivex.rxjava2:rxjava:$rxJavaVersion"
 
+    def iconifyVersion = "2.2.2"
     implementation "com.joanzapata.iconify:android-iconify-fontawesome:$iconifyVersion"
     implementation "com.joanzapata.iconify:android-iconify-material:$iconifyVersion"
-    implementation "com.yqritc:recyclerview-flexibledivider:$recyclerviewFlexibledividerVersion"
-    implementation "com.github.shts:TriangleLabelView:$triangleLabelViewVersion"
+    implementation 'com.yqritc:recyclerview-flexibledivider:1.4.0'
+    implementation 'com.github.shts:TriangleLabelView:1.1.2'
     implementation 'com.leinardi.android:speed-dial:3.1.1'
     implementation "com.github.AntennaPod:AntennaPod-AudioPlayer:$audioPlayerVersion"
     implementation 'com.github.mfietz:fyydlin:v0.5.0'
     implementation 'com.github.ByteHamster:SearchPreference:v2.0.0'
 
+    compileOnly "com.google.android.wearable:wearable:$wearableSupportVersion"
```

Comment:
Why is this now separated from the other dependencies?

Reply:
See https://github.com/AntennaPod/AntennaPod/pull/4120#discussion_r420494806. Was trying to match the app and core dependency list as close as possible.